vendor_consignment_stock/README.rst

```diff
@@ -0,0 +1,46 @@
+Vendor Consignment Stock
```

Comment:
Here is the template:
https://github.com/OCA/maintainers-tools/blob/master/template/module/README.rst